test/api.jl

```diff
@@ -209,7 +225,10 @@ B = [-1 0 0 1;
       0 0 1 0]
 Δ = [-1 0 0 0; 0 0 -1 -1; 0 -1 0 0; 0 0 0 0; 0 0 0 0]
 lcs = [[1,2,5],[3,4]]
-testnetwork(rns[5], B, Z, Δ, lcs, 1)
+r = reactions(rns[5])
+subrn = [[r[1],r[4],r[3]], [r[2]]  ]
```

Comment:
Just to double check; you calculated these by hand right, and are not just reusing what your code gives as the correct answer?

Reply:
> Just to double check; you calculated these by hand right, and are not just reusing what your code gives as the correct answer?

@isaacsas  i had checked them visually with the output of `complexgraph()` ,and see that the reaction tests match. Now with  https://github.com/SciML/Catalyst.jl/pull/396#discussion_r706517402 , since we are using `Set` , i will adjust `subrn` in `tests/api.jl` to be `Vector of Set of Reactions`.